devnet/node.yaml

```diff
@@ -75,6 +75,10 @@ spec:
             - ws://eth-devnet:8545
             - --avalancheRPC
             - ws://eth-devnet:8545
+           - --auroraRPC
+            - wss://testnet.aurora.dev
```

Comment:
I do not think we want to connect out to actual aurora testnet for this. For devnet (aka tilt) we are just using the same thing for all of them since they're all evm chains.

```suggestion
```